azure/storage/blob_test.go

```diff
@@ -50,7 +50,7 @@ func Test_blobSASStringToSign(t *testing.T) {
 }
 
 func TestGetBlobSASURI(t *testing.T) {
-	api, err := NewClient("foo", "YmFy", DefaultBaseUrl, "2013-08-15", true)
+	api, err := NewStorageClient("foo", "YmFy", DefaultBaseUrl, "2013-08-15", true)
```

Comment:
why is it NewClient in hostedservice package and NewStorageClient here? For consistency let's keem 'em NewClient because they'll always be preceded by package name anyway. i.e. storage.NewClient, virtualmachine.NewClient